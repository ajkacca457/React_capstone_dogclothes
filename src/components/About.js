import React from 'react';
import Aboutimg from '../images/aboutimg.png';

const About = () => (
  <div className="about">

    <div className="abtcont bg-light">
      <h3>About TVWORLD</h3>
      <img src={Aboutimg} alt="about" className="aboutimg my-4" />
      <p className="text-justify">
        Join the millions of subscribers around the world who enjoy unlimited award-winning
        TV shows, movies, documentaries, and more without a single advertisement.
        As a TVWORLD member, you are charged once a month on the date you signed up.
        There are no contracts, no cancellation fees, and no commitments.
        You have the freedom to change your plan or cancel online at any time if
        you decide TVWORLD isnâ€™t for you.
      </p>
    </div>

  </div>

);

export default About;
