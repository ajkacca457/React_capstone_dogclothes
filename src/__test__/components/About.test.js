import React from 'react';
import TestRenderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '../../components/About';

describe('About Component', () => {
  it('renders correctly', () => {
    const quotetree = TestRenderer
      .create(<About />)
      .toJSON();
    expect(quotetree).toMatchSnapshot();
  });

  test('renders the title text of About page', () => {
    const { getByText } = render(<About />);
    const title = getByText('About TVWORLD');
    expect(title).toBeInTheDocument();
  });
});
