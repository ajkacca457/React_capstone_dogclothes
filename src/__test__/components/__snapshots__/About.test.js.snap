// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component renders correctly 1`] = `
<div
  className="about"
>
  <div
    className="abtcont bg-light"
  >
    <h3>
      About TVWORLD
    </h3>
    <img
      alt="about"
      className="aboutimg my-4"
      src="aboutimg.png"
    />
    <p
      className="text-justify"
    >
      Join the millions of subscribers around the world who enjoy unlimited award-winning TV shows, movies, documentaries, and more without a single advertisement. As a TVWORLD member, you are charged once a month on the date you signed up. There are no contracts, no cancellation fees, and no commitments. You have the freedom to change your plan or cancel online at any time if you decide TVWORLD isnâ€™t for you.
    </p>
  </div>
</div>
`;
